js/content/community.js

```diff
@@ -410,14 +410,23 @@ let ProfileHomePageClass = (function(){
 
             let name =  HTML.escape(customLink.name);
             let link = "//" + HTML.escape(customUrl.replace("[ID]", steamId));
-            let icon = "//" + HTML.escape(customLink.icon);
+            let icon;
+            if (customLink.icon) {
+                icon = "//" + HTML.escape(customLink.icon);
+            } else {
+                iconType = "none";
+            }
 
             htmlstr +=
                 `<div class="es_profile_link profile_count_link">
                     <a class="es_sites_icons es_none es_custom_icon" href="${link}" target="_blank">
                     <span class="count_link_label">${name}</span>`;
                     if (iconType !== "none") {
-                        htmlstr += `<i class="es_sites_custom_icon" style="background-image: url(${icon});"></i>`;
+                        htmlstr += `<i class="es_sites_custom_icon" style="background-image: url(${icon});`
+                        if (iconType === "gray") {
+                            htmlstr += ` filter: grayscale(100%);`;
```

Comment:
I think this is already handled by [this class assignment](https://github.com/tfedor/AugmentedSteam/blob/58a4e3c7c09ddf25eaca5ddf55ae08aaa2229a40/js/content/community.js#L393)

Reply:
I think?

Reply:
Now I know

Reply:
Well, you were wrong. I just tested this. This line is required to make the custom icon greyscale.

Reply:
I also noticed this, but I thought this PR was about default profile links only (then I saw [this](https://github.com/tfedor/AugmentedSteam/issues/469#issuecomment-523214902)).

But for consistency that should also be solved via CSS classes